.github/workflows/publish.yml

```diff
@@ -18,5 +19,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install Dependencies
         run: npm install
-      - name: Publish Extension as Patch
+      - name: Publish Extension as Patch to marketplace
         run: npm run vsce:publish
+      - name: Publish Extension as Patch to open-vsx
+        run: ovsx publish -p "$OPEN_VSX_ACCESS_TOKEN"
```

Comment:
This should be `yarn ovsx` or `node_modules/.bin/ovsx`, `node_modules` binaries are not registered globally so this won't work afaik.